templates/travis/.travis/publish_client_gem.sh.j2

```diff
@@ -17,15 +17,17 @@ export DESCRIPTION="$(git describe --all --exact-match `git rev-parse HEAD`)"
 if [[ $DESCRIPTION == 'tags/'$REPORTED_VERSION ]]; then
   export VERSION=${REPORTED_VERSION}
 else
+  # Daily publishing of development version (ends in ".dev")
+  [ "${REPORTED_VERSION%.dev}" != "${REPORTED_VERSION}" ] || exit 1
   export EPOCH="$(date +%s)"
-  export VERSION=${REPORTED_VERSION}${EPOCH}
+  export VERSION=${REPORTED_VERSION}.${EPOCH}
```

Comment:
The test condition exits the entire script if the string does not end in .dev, correct?

So when we have a REPORTED_VERSION like:
3.0.0rc6.dev

Instead of it being listed as:
3.0.0rc6.dev1568383024
It will be listed as:
3.0.0rc6.dev.1568383024

Is that correct?

Because the example in [#5536](https://pulp.plan.io/issues/5536) does not list ".dev".

Reply:
Yes. The problem occurs, when it is _not_ ending in `.dev`.

Reply:
It turns out that the plugin version is reported as `"version": "2.0.0b3.dev0"` while it is specified `__version__ = "2.0.0b3.dev"` in the plugin.
This is in the source box; in the sandbox install this does not happen.

Reply:
@mdellweg So the source box installs nightly versions, and therefore the 0 gets appended to the letter at the end.

But the sandbox installs stable releases, which already has numbers at the end, so it appends nothing?